test/problems/second_quantization/molecular/test_molecular_problem.py

```diff
@@ -0,0 +1,71 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests Molecular Problem."""
+
+from test import QiskitNatureTestCase
+from qiskit_nature.problems.second_quantization.molecular.fermionic_op_builder import \
+    build_fermionic_op
+from qiskit_nature.transformers import ActiveSpaceTransformer
+from qiskit_nature.drivers import HDF5Driver
+from qiskit_nature.operators.second_quantization import SecondQuantizedOp
+from qiskit_nature.problems.second_quantization.molecular.molecular_problem import MolecularProblem
+
+
+class TestMolecularProblem(QiskitNatureTestCase):
+    """Tests Molecular Problem."""
+
+    def test_second_q_ops_without_transformers(self):
+        """Tests that the list of second quantized operators is created if no transformers
+        provided."""
+        expected_num_of_electr_terms = 185
+        expected_num_of_sec_quant_ops = 7
+
+        driver = HDF5Driver(hdf5_input=self.get_resource_path('H2_631g.hdf5', 'transformers'))
```

Comment:
I see here you have used HDF5Driver.... (I made the comment above before I saw this, but this is much better if a driver is needed as an integrals source for the test.)